pkg/sbom/core/bom.go

```diff
@@ -126,6 +127,10 @@ type Component struct {
 	// SPDX: package.licenseConcluded, package.licenseDeclared
 	Licenses []string
 
+	// LicensesV2 is a list of concluded licenses that apply to the component
+	// SPDX: package.licenseConcluded
+	LicensesV2 []types.License
```

Comment:
Any reason for naming it LIcensesV2 and not ConcludedLicenses ?

Reply:
earlier i used it store declared + concluded licenses as part of licesnses v2, but now we only store concluded licenses in it. we can change if it's needed, what do you say? @john-d8r @theakshaymenon